lib/logitech_receiver/settings.py

```diff
@@ -140,7 +142,7 @@ def write(self, value):
 			data_bytes = self._validator.prepare_write(value, current_value)
 			if data_bytes is not None:
 				if _log.isEnabledFor(_DEBUG):
-					_log.debug("%s: prepare write(%s) => %r", self.name, value, data_bytes)
+					_log.debug("%s: Settings prepare write(%s) => %r", self.name, value, data_bytes)
```

Comment:
nitpick: lowercase (`settings`)

Reply:
I force-pushed a commit with the 3 Settings turned into settings.